packages/pds/src/db/migrations/20230411T175730759Z-drop-message-queue.ts

```diff
@@ -0,0 +1,32 @@
+import { Kysely } from 'kysely'
+import { Dialect } from '..'
+
+const messageQueueTable = 'message_queue'
+const messageQueueCursorTable = 'message_queue_cursor'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
```

Comment:
We spoke about potentially wanting to split off this migration into a separate PR.